cli/src/index.ts

```diff
@@ -55,6 +58,12 @@ const main = async () => {
   ) as CT3APackageJSON;
   pkgJson.name = scopedAppName;
   pkgJson.ct3aMetadata = { initVersion: getVersion() };
+
+  const { stdout } = await execa(pkgManager, ["-v"], {
+    cwd: projectDir,
+  });
+  pkgJson.packageManager = `${pkgManager}@${stdout.trim()}`;
```

Comment:
We should not do this for Bun since its not supported in this field (yet)

Reply:
Done, added a comment for future reference.